env/version.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2020 Autodesk, Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Autodesk, Inc.
+#
+
+description: Toolkit configuration to be loaded whenever an association with a
+             Version has been established.
+
+includes:
+- includes/houdini/site.yml
+- includes/maya/site.yml
+- includes/nuke/site.yml
+# - includes/flame/site.yml
+- includes/desktop/site.yml
+- includes/desktop2/all.yml
+- includes/shell/site.yml
+- includes/photoshopcc/site.yml
+- includes/aftereffects/site.yml
+- includes/3dsmax/site.yml
+- includes/alias/site.yml
+- includes/shotgun/all.yml
+- includes/vred/site.yml
+
+engines:
+  tk-3dsmax: '@3dsmax.site'
+  tk-3dsmaxplus: '@3dsmaxplus.site'
+  tk-aftereffects: '@aftereffects.site'
+  tk-alias: '@alias.site'
+  tk-desktop: '@desktop.site'
+  tk-desktop2: '@desktop2.all'
+  # tk-flame: '@flame.site'
+  tk-houdini: '@houdini.site'
+  tk-maya: '@maya.site'
+  tk-nuke: '@nuke.site'
+  # tk-nukestudio: '@nukestudio.site'
+  tk-photoshopcc: '@photoshopcc.site'
+  tk-shell: '@shell.site'
+  tk-shotgun: '@shotgun.version'
+  tk-vred: '@vred.site'
```

Comment:
All of the `site` references should actually be pointing to `project` settings. This is what the old config was falling back to. 

It would also be good to leave a comment somewhere in this file to highlight the reason it was created even though it looks almost identical to the `project.yml`. ie: It was added to allow us to add Version specific actions for VRED, but we still want to maintain the old functionality elsewhere.

Reply:
Makes sense -- fixed.